pyproject.toml

```diff
@@ -21,14 +21,16 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6.2"
-diffsync = "^1.3.0"
-cloudvision = "^1.0.0"
+python = "^3.7"
+nautobot = "^1.2.0"
 nautobot-ssot = "^1.0.1"
+Jinja2 = "<3"
```

Comment:
This will prevent usage with Nautobot 1.3.0 and later which require Jinja2 3.x.

Reply:
Updated